### PR TITLE
New angular

### PR DESCRIPTION
**Pull request by bobziroll**
I'm pretty sure there are no bugs and the variable names are PERFECT. But you wanted to look at these anyway, even though I'm the best.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,6 +2,22 @@ var app = angular.module("MyApp", []);
 
 app.controller("MainController", ["$scope", function($scope) {
 
-    $scope.test = "Hey";
-
+    $scope.ps = [
```
Comment (joevus) [inline app.js]: PS, what the heck is ps? Please give this a better variable name.

File: app.js [hunk 2]
```diff
@@ -2,6 +2,22 @@ var app = angular.module("MyApp", []);
...
+        {
+            fn: "Bob",
```
Comment (joevus) [inline app.js]: please choose another property name.

File: index.html [hunk 3]
```diff
@@ -9,7 +9,9 @@
 </head>
 
 <body ng-controller="MainController">
-    <h1>{{ test }}</h1>
+    <ul>
+        <li ng-repeat="p in ps">{{ p.fN }}</li>
```
Comment (joevus) [inline index.html]: It should be fn with all lowercase, but you should rename this anyway.

File: app.js (hunk 1 above)
Comment (bobziroll) [inline app.js]: FINE

File: app.js (hunk 2 above)
Comment (bobziroll) [inline app.js]: GEEZ OKAY

File: app.js [hunk 4]
```diff
@@ -2,6 +2,22 @@ var app = angular.module("MyApp", []);
 
 app.controller("MainController", ["$scope", function($scope) {
 
-    $scope.test = "Hey";
-
+    $scope.people = [
+        {
+            firstName: "Bob",
```
Comment (joevus) [inline app.js]: looking better. you're fired.

**Pull request merged**